lib/marked.cjs

```diff
@@ -28,6 +28,20 @@ function _createClass(Constructor, protoProps, staticProps) {
   });
   return Constructor;
 }
+function _extends() {
```

Comment:
What's the deal with `_extends` here? This appears to be replacing `merge()` but I thought #2730 already replaced that with the spread operator.

Also why are the bundled files appearing in this PR?

Reply:
#2730 hasn't been merged yet. I included the lib folder so I could test the demo with this PR. The lib folder is rebuilt when released so it shouldn't matter if it is included in a PR.

I think the `_extends` method is added by roll-up.